redaxo/src/addons/cronjob/install.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+rex_sql::factory()->setQuery('
+    CREATE TABLE IF NOT EXISTS `'.rex::getTable('cronjob').'` (
+        id int(10) unsigned NOT NULL AUTO_INCREMENT,
+        PRIMARY KEY (`id`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+');
+
+$table = rex_sql_table::get(rex::getTable('cronjob'));
```

Comment:
ich denke wir sollten hier "versions-kommentare" drann machen, damit wir sehen wann was dazugekommen ist.

aktuell könnte ein "leser" des codes sich fragen warum nicht in einem schritt der "zielzustand" erreicht wird.

alternativ, file-kommentar oben ins file